app/controllers/admin/basic_projects_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
 
     @csv_mode = true
 
-    export_grid_if_requested do
+    export_grid_if_requested('basic_projects': 'basic_projects_grid_definition') do
```

Comment:
This is kind of yucky boilerplate but I guess wice_grid needs it and it's probably not worth creating a helper method to simplify it.